src/map/Map.js

```diff
@@ -16,7 +16,9 @@ L.Map = L.Evented.extend({
 		fadeAnimation: true,
 		trackResize: true,
 		markerZoomAnimation: true,
-		maxBoundsViscosity: 0.0
+		maxBoundsViscosity: 0.0,
+		zoomSnap: 1,

```

Comment:
Any operation that changes zoom level (including fitBounds and stopping a flyTo) will snap the zoom level to be a multiple of `map.options.zoomSnap`.